ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs

```diff
@@ -0,0 +1,55 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Breadcrumbs::Item
+  @text={{t 'resources.alias.title'}}
+  @route='scopes.scope.targets.target.manage-alias'
+/>
+<Rose::Layout::Page as |page|>
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+  <page.header>
+    <h2>
+      {{t 'resources.alias.title'}}
+      {{! TODO: DOC LINK }}
+    </h2>
+    <p>{{t 'resources.alias.description'}}</p>
+    <Hds::Copy::Snippet @color='secondary' @textToCopy={{@model.id}} />
+  </page.header>
+  <page.actions>
+
+    <Hds::Dropdown as |dd|>
+      <dd.ToggleButton
+        data-test-manage-dropdown
+        @text={{t 'actions.manage'}}
+        @color='secondary'
+      />
+
+      <dd.Interactive
+        @text={{t 'actions.clear'}}
+        {{on 'click' (route-action 'clearAlias' @model)}}
```

Comment:
note: I did not move these to controllers as the target route involves quite a bit of changes

Reply:
I was going to suggest for the sake of the code freeze to leave the refactor to controllers for later. We already have a [jira task](https://hashicorp.atlassian.net/browse/ICU-12611) dedicated to this, so all good 😉

Reply:
thanks, for linking that, I was not aware that we already had tickets for this :D